images-list-daily

```diff
@@ -2,5 +2,13 @@
 # See images-list for more information
```

Comment:
I only checked a few, but those were already in the origin images, if you can confirm the rest: https://github.com/rancher/rancher/blob/release/v2.7/pkg/image/origins.go

Reply:
I can confirm that they are all in the original images list. 
Actually, all entries added in this PR already exist in [images-list](https://github.com/rancher/image-mirror/blob/master/images-list)